### DISCUSSION TITLE
Chaotic Aur

### DISCUSSION THREAD

**Discussion by se7en-x230** [kRHYME7/Hyde-cli#14] [Q&A]
I see in the scrips folder.
How do I enable chaoticAur?

**Comment by kRHYME7** [comment]
I only enabled that upon installation of hyprdots. 
But you can run ``` /usr/lib/hyprdots-ctl/Install_ChaoticAUR  --intall ``` 

Warning: 
~Based on experience if you use a reflector or any package that alters your mirrors, You need to rerun the installation of chaotic AUR. I think the reflector overwrites some chaotic AUR lines.~
Reflector Actually fixes it.

**Reply by dieBakterie** [reply]
I've installed paru to Install hyprdots-ctl-git and now it's using aur and not chaotic aur

**Reply by kRHYME7** [reply]
I already told you paru is not related with chaotic AUR. 😿

This is the repo for chaotic AUr https://aur.chaotic.cx/ 

paru is the same with yay just written on another language. 

if you are interested on using chaotic the command is ``` Hyprdots inject chaotic --install fresh ```

**Reply by dieBakterie** [reply]
Thanks and sorry again for this

**Reply by kRHYME7** [reply]
No probs.

**Reply by dieBakterie** [reply]
Sorry but IT seems your cmd isnt working or i'm dumb?

**Reply by kRHYME7** [reply]
Can you provide the output ? 

In case just update the CLI ``` paru -Sy hyprdots-ctl-git ```

**Reply by dieBakterie** [reply]
![IMG_20240325_145423.jpg](https://github.com/kRHYME7/Hyprdots-ctl/assets/50206827/ba8501ca-5985-4515-a6b1-9a320328280f)

That's the output

**Reply by kRHYME7** [reply]
![image](https://github.com/kRHYME7/Hyprdots-ctl/assets/53417443/28a69a34-d7d9-4f5c-8662-7534faca9a40)

How about running ``` Hyprdots ``` alone?

Oh no Base on the error you are not using the latest commit. Or this directory does not exist
![image](https://github.com/kRHYME7/Hyprdots-ctl/assets/53417443/7cef63b3-3c25-4024-a86b-0bfcdf6465c6)

Have you tried updating since then ?

``` ~/.config/hypr/scripts/*```  is now at ./hyprdots  ``` ~/.config/hyprdots/scripts/*```

**Reply by dieBakterie** [reply]
Yeah

**Reply by kRHYME7** [reply]
Output for this ? 

```
ls ~/.config/hyprdots/scripts/globalcontrol.sh
ls ~/Hyprdots/Scripts/global_fn.sh
```

There's a big chance you tried to install another repo but did not populate or link some files successfully
Let me help you solve it.

**Reply by dieBakterie** [reply]
The second one exists the First arent

**Reply by kRHYME7** [reply]
I mean the Hyprdots repo. 

what about the output?

```
ls ~/.config/hyprdots/scripts/globalcontrol.sh
ls ~/Hyprdots/Scripts/global_fn.sh

```

**Reply by kRHYME7** [reply]
```  ~/.config/hyprdots/scripts/globalcontrol.sh ``` 

Therefore this one is not?

How about this?
``` ls ~/.config/hypr/scripts/globalcontrol.sh ```

**Reply by dieBakterie** [reply]
> This don't exists
> 
> ```
> ls ~/.config/hyprdots/scripts/globalcontrol.sh
>
> ```

**Reply by dieBakterie** [reply]
In `.config` is only the `systemd` folder

**Reply by kRHYME7** [reply]
> In .configs IS only the systemd folder

I don't understand this one

But for sure, you did not install it right!

Read this first because your configurations are already deprecated. 

https://github.com/prasanthrangan/hyprdots/issues/1053

**Reply by dieBakterie** [reply]
Yeah it's a fresh install and when it asked to install i said no cause fought remove aur and redo to skip manual chaotic setup. Sorry for the trouble

**Reply by kRHYME7** [reply]
Yeah, no probs! I've updated it already, so user can skip the prompt on installing chaotic aur. Please don't hesitate to ask, but do it in the discussions panel not on the issue tracker.

**Reply by dieBakterie** [reply]
Yeah Sure and thanks mate
